lib/pulfalight/traject/ead2_component_config.rb

```diff
@@ -296,9 +296,15 @@
     container_location_code = container_element.attributes["altrender"].to_s
     container_profile = container_element.attributes["encodinganalog"].to_s
     next if container_location_code.blank?
+    barcode_label = container_element.attributes["label"].to_s
+    barcode_match = barcode_label.match(/\[(\d+?)\]/)
```

Comment:
I am surprised that there are ASpace container barcodes which would not match this.